modules/vsphere-problem-detector-config-checks.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/using-vsphere-problem-detector-operator.adoc
+
+:operator-name: vSphere Problem Detector Operator
+
+[id="vsphere-problem-detector-config-checks_{context}"]
+= Configuration checks run by the vSphere Problem Detector Operator
+
+The following tables identify the configuration checks that are run by the {operator-name}. Some checks verify the configuration of the cluster.
+Other checks verify the configuration of each node in the cluster.
+
+.Cluster configuration checks
+[options="header",cols="20,80a"]
+|===
+|Name
+|Description
+
+|CheckDefaultDatastore
+|Verifies that the default datastore name in the vSphere configuration is short enough to be used with dynamic provisioning.
+See xref:storage-class-configuration-check_{context}[About the storage class configuration check] for more information about the character length limit for the datastore name, persistent volume names, and systemd.
+
+If this check fails, you should expect the following:
+
+* systemd logs errors to the journal such as `Failed to set up mount unit: Invalid argument`.
```

Comment:
Service names need to be enclosed in backticks. See https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

Reply:
My mistake. I noticed that systemd frequently didn't have backticks in a number of places such as this one: https://docs.openshift.com/container-platform/4.6/architecture/control-plane.html